.changeset/beige-kiwis-begin.md

```diff
@@ -0,0 +1,6 @@
+---
+'@urql/exchange-graphcache': minor
+---
+
+Issue warnings when an unknown query, type or field has been included in Graphcache's opts.resolvers configuration to help spot typos.
+See: [#820](https://github.com/FormidableLabs/urql/pull/820)
```

Comment:
I’d remove this since we’ll already have the other changeset pointing to this previous PR :)
The reference should be added automatically. There’s also two more duplicate changesets I believe

Reply:
Changesets merged into `.changeset/thick-avocados-enjoy.md`